sdk/identity/identity/samples/AzureCacheForRedis/node-redis.md

```diff
@@ -0,0 +1,276 @@
+## Azure Cache for Redis: Azure AD with node-redis client library
```

Comment:
I _really_ think that we should just pick one. This package's API is so, so nearly the _exact same thing_ as ioredis. I just don't see much value at all in having both of these samples that say the exact same thing. So much of the code is just duplicate information and which library you use to talk to redis is beside the point of the sample, which is about how Azure Cache for Redis interacts with AAD RBAC.